export/famplex.obo

```diff
@@ -15,6 +15,7 @@ inverse_is_a: HGNC:HTR7
 [Term]
 id: FPLX:5_hydroxytryptamine_receptors_ionotropic
 name: 5-hydroxytryptamine-receptors-ionotropic
+def: "A subclass of serotonin receptors that form cation channels and mediate signal transduction by depolarizing the cell membrane. The cation channels are formed from 5 receptor subunits. When stimulated the receptors allow the selective passage of SODIUM; POTASSIUM; and CALCIUM." [mesh:D044406]
```

Comment:
oh crazy, somehow I forgot to update the OBO file when I made the PR that added the definitions.

Reply:
I was just wondering how come the tests didn't error last time, they should normally error on any diffs in exports...